water-box/requirements.txt

```diff
@@ -0,0 +1,5 @@
+numpy
+mbuild
+foyer
+mdtraj
+matplotlib
```

Comment:
What about gromacs? 

Also, there is no requirements.txt under the water-models tutorial. 

And both water-box and water-models examples require openbabel  to run.

Reply:
I added installation instructions in `water-models.ipynb` and added `requirements.txt` to that example as well.  Gromacs can be installed from the bioconda channel and open babel can be installed from the open babel channel.